system-int-test/test/cmr/system_int_test/search/collection_spatial_search_test.clj

```diff
@@ -441,6 +433,23 @@
         ["0,89,100" "0,89,1000000"] [whole-world on-np]
         ["0,0,1000" "0,89,1000" "0,89,1000000"] [whole-world]))
 
+    (testing "valid circle searches with ORed results"
+      (are [lon-lat-radius items]
+        (let [found (search/find-refs
+                     :collection
+                     {:circle lon-lat-radius
+                      :page-size 50
+                      "options[spatial][or]" "true"})
+              matches? (d/refs-match? items found)]
+          (when-not matches?
```

Comment:
Are the `println` invoked if the assertion fails?

Reply:
Yes, it's just the test output that gives the expected vs actual